frontend/src/pages/MatchPage.tsx

```diff
@@ -1,61 +1,61 @@
 import React, { useEffect, useMemo, useState } from 'react';
 import { Button, CircularProgress } from '@mui/material';
 import { io, Socket } from 'socket.io-client';
+import { useCookies } from 'react-cookie';
 import { useAuthContext } from '../contexts/AuthContext';
+import { Difficulty, Match, SocketEvent } from '../constants';
+import { ConnectedElseWhereModal, AlreadyInQueueModal } from '../components';
 
 export default () => {
-  const socket = useMemo<Socket>(() => io('http://localhost:8001'), []);
+  const [cookie] = useCookies(['userCred']);
```

Comment:
Actually my authcontext got cookies alrd. So can just import and use. e.g.  const { user, cookie, setCookie } = useAuthContext();